src/create-boilerplate.ts

```diff
@@ -0,0 +1,71 @@
+import { PromptObject } from 'prompts';
+import { readApiData, writeApiData, promptQuestions } from './utils/filesystem';
+import { runAndHandleErrors } from './utils/cli';
+import { emptyObject } from './utils/normalization';
+import { Api, ApiMetadata, Beacons, Oises, Templates } from './types';
+
+const questions: PromptObject[] = [
+  {
+    type: 'text',
+    name: 'apiName',
+    message: 'What is the name of the API Integration?',
```

Comment:
Nice :)
You can add an "initial" field here which visually prompts the user with an example of the data being asked for.
An example is here: https://github.com/api3dao/beacon-setup-guide/blob/main/scripts/setup-integration.ts